spec/views/batch_contexts/show.html.erb_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 RSpec.describe 'batch_contexts/show.html.erb' do
-  let(:bc) { create(:batch_context) }
+  let(:bc) { create(:batch_context_with_deleted_output_dir) }
```

Comment:
This test could not be run multiple times because the first run would create an output dir and the second batch context create would bomb because the output dir exists. This uses a factory which deletes the output dir.